pkg/controller.v1/tensorflow/job_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	tfv1 "github.com/kubeflow/training-operator/pkg/apis/tensorflow/v1"
+	kubeflowv1 "github.com/kubeflow/training-operator/pkg/apis/kubeflow.org/v1"
```

Comment:
For clarity: Is adding suffix(.org)  the standard convention(Say, instead of `kubeflow` or `training`) ? Similarly I see the same in SDK, `KubeflowOrgV1` ?

Reply:
Yes.
According to [this code-generator document](https://github.com/kubernetes/code-generator/blob/65c70a51ae8eca63d4b9ab1e80fe636097a6ce63/generate-groups.sh#L31), we need to specify group name on it.
So, I renamed the directory to `pkg/apis/kubeflow.org/v1`.

The same directory structure is used for [`kubernetes-sigs/autoscaler/vertical-pod-autoscaler`](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/pkg), [`rook/rook`](https://github.com/rook/rook/tree/master/pkg), etc.

What do you think? @johnugeorge

Reply:
Thanks for explaining. 

LGTM